docs/faq.md

```diff
@@ -81,7 +81,8 @@ We list some common troubles faced by many users and their corresponding solutio
 
 - "RuntimeError: Expected to have finished reduction in the prior iteration before starting a new one"
     1. This error indicates that your module has parameters that were not used in producing loss. This phenomenon may be caused by running different branches in your code in DDP mode.
-    2. You can set ` find_unused_parameters = True` in the config to solve the above problems or find those unused parameters manually.
+    2. You can set ` find_unused_parameters = True` in the config to solve the above problems(but this will slow down the training speed.
+    3. If the version of your MMCV >= 1.4.1, you can get the name of those unused parameters with `detect_anomalous_params=True` in `OptimizerHook` then delete them.
```

Comment:
Should change to set `detect_anomalous_params=True` in `optimizer_config` instead of `OptimizerHook` because there is no `OptimizerHook` key in the config.